src/main.rs

```diff
@@ -44,22 +47,24 @@ type ServerRef = Arc<Server>;
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let configuration: Settings = Settings::new().expect("couldn't load config");
+
     let mining_settings = MiningSettings {
         local_work_size: 256,
-        kernel_size: 1 << 22,
```

Comment:
Make the 22 configurable as "intensity" and the gpu_indices configurable as "gpu_index"

Reply:
Was going to do that in a followup PR, including this particular thing was a mistake. Can do it in the next update if needed.